core/src/utilities/helpers/generic-helpers.js

```diff
@@ -400,6 +400,28 @@ class GenericHelpersClass {
   isString(value) {
     return typeof value === 'string' || value instanceof String;
   }
+
+  removeKeyValueFromURL(urlObj, keyName) {
```

Comment:
Would be nice to have a unit test on this function. I for myself am not able to read all possible scenarios on this one.
(If we change this function later to something more simple its functionality should be the same, so won't need to re-write the test again).

 (Maybe to make the test easier just get **LuigiConfig.getConfigBooleanValue('routing.useHashRouting');** out of it and pass it as a parameter so you don't have to do extra mocks)